draft-opsarea-rfc5706bis.md

```diff
@@ -563,6 +568,30 @@ This document does not describe interoperability requirements. As such, it does
    possible to generate the operations necessary to get from A to B with
    minimal state changes and effects on network and systems.
 
+## Impact on Security Operations {#sec-impact-secops}
+
+   Security Operations (SecOps) is a collaborative approach that combines security and operational teams to protect networks and digital assets {{SECOPS}}. Historically, these functions were performed separately, with siloed teams having different priorities. SecOps combines these roles to improve the ability of operators to protect and manage the network effectively and efficiently. Security operators detect malicious activity and respond to threats and are a crucial part of defending against attacks alongside the management and operation of the network.  
+
+   Protocol Designers should consider the impacts of a New Protocol or Protocol Extension on Security Operations (SecOps) in networks that the protocol will be deployed in. 
+
+   Indicators of Compromise (IoCs) {{?RFC9424}} are relied upon by security operators to identify and block malicious activity. A New Protocol or Protocol Extension may have implications for the types, locations or availability of IoCs and Protocol Designers should outline such impacts. Similarly, to be most effective, IoCs are required to be shared at scale so consideration should be given to ensure this is possible. 
```

Comment:
Maybe an example of implications and what "at scale" in this case means would be helpful.